clarifai/client/client.py

```diff
@@ -592,7 +592,7 @@ def _setup_multi_data(self, ops, num_cases, model=None, local_ids=None, meta=Non
     """ Setup the data dict to POST to the server. """
     data =  {'op': ','.join(ops)}
     if model:  # use the variable passed into method
-      data['model'] = self._sanitize_param(model, 'default')
+      data['model'] = self._sanitize_param(model)
```

Comment:
In case the param has bad unicode, the client should just raise an error, rather than silently switching to a different model.  And i'm not 100% convinced that the server will treat an empty string the same as not sending a model param.